### DISCUSSION TITLE
[A/L] (2021/22) Logica - Esercizio 29

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#355] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152326031-da5ae65e-fb9b-4132-81d7-52a660dd421a.png)

**Comment by francesco-alberghina** [comment]
Questa è la mia soluzione:
![Screenshot 2024-06-19 173623](https://github.com/sapienzastudentsnetwork/mmi2122/assets/68157574/6118c07f-8788-41f9-ac36-88a8d6e200c1)
